src/runtime_src/hip/core/event.h

```diff
@@ -62,8 +62,7 @@ class command
   state get_state() const { return cstate; }
   std::chrono::time_point<std::chrono::system_clock> get_time() { return ctime; }
   void set_state(state newstate) { cstate = newstate; };
-  type
-  get_type() const { return ctype; }
+  type get_type() const { return ctype; }
```

Comment:
This is where I complete disagree with @keryell, I have to scan really really hard to find the functions far to the right, e.g. get_time().  It is much more preferable to format as
```
ReturnType
function() 
{}
```
where everything is on the left margin.

Reply:
Thanks, I modified the functions thoroughly.